app/src/main/java/com/github/sdp_begreen/begreen/firebase/FirebaseDB.kt

```diff
@@ -293,4 +301,40 @@ object FirebaseDB {
             null
         }
     }
+
+    /**
+     * Store the given [location] as the location of a recycling bin.
+     *
+     * @param location the location of the bin
+     *
+     * @return true if the location got stored in the database, and false if it failed
+     */
+    suspend fun storeBinLocation(location: LatLng): Boolean{

```

Comment:
Great ideas! I actually thought about adding up and down votes, such that a bin gets removed if it has a too low ratio of up to down votes (to prevent some mailcious user from deleting every markers). I think we can add this feature once google maps is mergde into the app.

Reply:
Good idea. I will take it into consideration during my work.